scene/src/components/backpack/WearableCategoryButton.tsx

```diff
@@ -0,0 +1,125 @@
+import { UiEntity, type UiTransformProps } from '@dcl/sdk/react-ecs'
+import { type WearableCategory } from '../../service/wearable-categories'
+import ReactEcs, {
+  type ReactElement,
+  type UiBackgroundProps
+} from '@dcl/react-ecs'
+import { getCanvasScaleRatio } from '../../service/canvas-ratio'
+import { Color4 } from '@dcl/sdk/math'
+import { COLOR } from '../color-palette'
+import Icon from '../icon/Icon'
+import type { AtlasIcon, URNWithoutTokenId } from '../../utils/definitions'
+import { noop } from '../../utils/function-utils'
+import { getBackgroundFromAtlas } from '../../utils/ui-utils'
+import { catalystWearableMap } from '../../utils/wearables-promise-utils'
+import { ROUNDED_TEXTURE_BACKGROUND } from '../../utils/constants'
+
+type WearableCategoryButtonProps = {
+  uiTransform?: UiTransformProps
+  category: WearableCategory
+  active?: boolean
+  onClick?: () => void
+  selectedURN: URNWithoutTokenId | null
+}
+export function WearableCategoryButton({
+  category,
+  uiTransform,
+  active,
+  onClick,
+  selectedURN
+}: WearableCategoryButtonProps): ReactElement {
+  const callbacks: { onClick: () => void } = {
+    onClick: onClick ?? noop
+  }
+  const canvasScaleRatio = getCanvasScaleRatio()
+  const categoryIcon: AtlasIcon = {
+    spriteName: category,
+    atlasName: 'backpack'
+  }
+  const iconSize = 34 * canvasScaleRatio * 2
+  const thumbnailSize = iconSize * 1.7
+
+  const textureProps: UiBackgroundProps =
+    selectedURN === null
+      ? getBackgroundFromAtlas({
+          atlasName: 'backpack',
+          spriteName: 'nft-empty'
+        })
+      : {
+          texture: {
+            src: `https://peer.decentraland.org/lambdas/collections/contents/${selectedURN}/thumbnail`
+          },
+          textureMode: 'stretch'
+        }
+
+  return (
+    <UiEntity
+      uiTransform={{
+        ...uiTransform,
+        width: 124 * canvasScaleRatio * 1.9,
+        height: 70 * canvasScaleRatio * 1.9,
+        margin: {
+          left: 10 * canvasScaleRatio * 2,
+          bottom: 6 * canvasScaleRatio * 2
+        },
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'space-around',
+        padding: 2 * canvasScaleRatio * 2
+      }}
+      uiBackground={{
+        ...ROUNDED_TEXTURE_BACKGROUND,
+        color:
+          active === true
+            ? COLOR.ACTIVE_BACKGROUND_COLOR
+            : Color4.create(0, 0, 0, 0.2)
+      }}
+      onMouseDown={() => {
+        callbacks.onClick()
+      }}
```

Comment:
I think this can be done, and looks simpler, without the need of the `callbacks` variable

```suggestion
  onMouseDown={() => (onClick ?? noop)()}
```

Reply:
https://github.com/dclexplorer/bevy-ui-scene/commit/5ccb06305414633916aff1b02c6259eba457da45